packages/components/Button/src/Button.android.tsx

```diff
@@ -0,0 +1,82 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { Pressable, View } from 'react-native';
+import { IButtonSlotProps, IButtonState, IButtonProps, IButtonRenderData, buttonName, IButtonType } from './Button.types';
+import { compose, IUseComposeStyling } from '@uifabricshared/foundation-compose';
+import { ISlots, withSlots } from '@uifabricshared/foundation-composable';
+import { Text } from '@fluentui-react-native/text';
+import { settings } from './Button.settings';
+import { backgroundColorTokens, borderTokens, textTokens, foregroundColorTokens, getPaletteFromTheme } from '@fluentui-react-native/tokens';
+import { filterViewProps } from '@fluentui-react-native/adapters';
+import { mergeSettings } from '@uifabricshared/foundation-settings';
+import { useAsPressable, useViewCommandFocus } from '@fluentui-react-native/interactive-hooks';
+import { Icon } from '@fluentui-react-native/icon';
+import { createIconProps } from './Button.icon';
+
+export const Button = compose<IButtonType>({
```

Comment:
I also wonder if we should be making these modfications to `experimental-button` since that uses the new compose framework, and seems to be the direction we want to go when I asked. 
I personally couldn't get styling to work properly on the experimental button with the apple theme, so the apple them mostly works well with this button.

Reply:
Will try if I could get `experimental-button` to work in the same PR.